app/controllers/EmailController.java

```diff
@@ -19,13 +20,17 @@
 @Controller("emailController")
 public class EmailController extends BaseController {
 
+    private BridgeConfig bridgeConfig;
```

Comment:
This is potentially dangerous. This bridgeConfig is actually different from BaseController.bridgeConfig, but we override the setter, so weird things could happen. If the subclass needs access to this, I strongly recommend using a getBridgeConfig() instead.